### PR TITLE
[Feature] GCKey Sign In Page

### LINKED ISSUE #7696: ✨ Sign in using GCKey
## ✨ Sign in using GCKey

This page will replace the [current GCKey sign in page](https://talent.canada.ca/en/login-info). 

The idea is that this is give the users more context and aids before we send them off to GCKey.

## 🎨 Figma file
[Design link](https://www.figma.com/file/GGyd3OuIpYxHBydvx65Fk2/Candidates---GCKey-prep-pages?type=design&node-id=109-7445&mode=design&t=5WFdEmxhChielXb2-4)

### 🧑‍🎨 Designer

@BrookeCameron 

## 📸 Screenshot
Screenshot 2023-08-25 at 9 12 49 AM

### 🧩 Component 
Regular page header with graphic 

### ✍🏼 Copy 
H1: Sign in using GCKey 

### ✍🏼 Copy 
H2: Welcome back
Paragraph: Just a reminder you’ll be leaving our site to sign in with GCKey. If anything goes wrong, we have prepared additional guidance for you. 

### 🎯 Button 
Button copy: Continue to GCKey and sign in
Button link: https://clegc-gckey.gc.ca/j/eng/l?ReqID=_1a539bc49ac8d7c8b2d4 

### ✍🏼 Copy 
H3: Signing in to your account
H4: Steps to signing in with GCKey and completing two-factor authentication 

#### Step 1
Image: 
Gckey_illustration_blocks_V1_Sign in
Image alt: mark as decorative 
Paragraph copy: 1. Sign in with your username and password. Remember, **your username is separate from your email address.** 

#### Step 2
Image: 
Gckey_illustration_blocks_V1_Connected auth app
Image alt: mark as decorative 
Paragraph copy: 2. **Open the authenticator app** on your device.

#### Step 3
Image: 
Gckey_illustration_blocks_V1_6 digit code copy
Image alt: mark as decorative 
Paragraph copy: 3. Enter your **unique one-time six-digit** code from your **authenticator app** into the verification bar.

#### Step 4
Image: 
Gckey_illustration_blocks_V1_Hooray copy
Image alt: mark as decorative 
Paragraph copy: 4. Hooray! **You’ve signed in with GCKey** and will be returned to the G**C Digital Talent platform.**

### 🧩 Component 
Expansion block simple

### ✍🏼 Copy 

**What if I deleted the app or changed phone and I don’t have the recovery codes?**
Although your login cannot be recovered, you can contact our [Help Desk](https://talent.canada.ca/en/support), and they can help you recover your account.

**Can you remove the two-factor authentication from my account so I can reset it?**
We cannot remove the two factor authentication from your account, but you can contact our [Help Desk](https://talent.canada.ca/en/support) and they can assist you with account recovery.

**What can I do if my authenticator codes are no longer being accepted?**
Please contact our [Help Desk](https://talent.canada.ca/en/support), and they can help you recover your account.

**Already have a GCKey account?**
If you already have a GCKey account you can sign in to your GC Digital Talent profile using your existing GCKey, even if you've never used this platform before.
If you're unsure whether you have an existing GCKey account, continue to the website and try signing in. If you can't remember your password, you can also reset it there.

**What is a GCKey?**
A GCKey is a central credential not managed by the GC Digital Talent team. The Government of Canada offers it as a way for you to communicate securely with many online-enabled Government programs and services.

**Who do I contact if I have questions about GCKey?**
If you have questions about GCKey, please contact the GCKey team at:
Canada and the United States
1-855-438-1102
Text Telephone (TTY/TDD)
1-855-438-1103
Outside Canada and the United States
1-800-2318-6290
Customer Service Representatives are available to assist you by phone,
year round, 24 hours a day, 7 days a week.

**Which authenticator app should I install?**
As the Government of Canada we cannot recommend any specific third-party vendors or apps. Well known digital vendors, like Google Authenticator and Microsoft Authenticator, provide authenticator apps. Whichever app you choose, ensure that it comes from a reputable vendor. 

**Can I use SMS or email authentication instead of an app?**
Currently, our site only supports authentication through an authenticator app.

Read all the FAQ’s and still stuck?  [Contact our team for help](https://talent.canada.ca/en/support)

## bottom of page buttons 

[Sign up instead](https://talent.canada.ca/en/register-info) 

[Continue to GCKey and sign in](https://clegc-gckey.gc.ca/j/eng/l?ReqID=_7d9e75dbf7bf2ee0af67)

## ✅ Acceptance Criteria

- [ ] Page matches Figma design
- [ ] Page exists an expected URL
- [ ] Storybook story
- [ ] Review from key designer

### PR DESCRIPTION
**Pull request by JamesHuf**
🤖 Resolves #7696 

## 👋 Introduction

This branch includes a new look for the Sign In page.

## 🕵️ Details

- The special IAP view still exists and has not been modified.  You may want to review the diff with whitespace changes hidden.
image

- The steps should use a proper ordered list but I wasn't able to get it working with images for each step. We should probably see about updating this in the future.

## 🧪 Testing

Assist reviewers with steps they can take to test that the PR does what it says it does.

1. Rebuild the site
2. Visit `http://localhost:8000/en/login-info`

## 📸 Screenshot

image

### CODE REVIEW

**Comment by BrookeCameron** [comment]
Overall this is looking really great. Here are my comments: 

1.  The icons are a bit on the thin side. These should be around 3-4pts thick. 
2. The spacing on this page seems tight - please add more white space between each section (should be the 1.5x space variable) 
3. Headings are the wrong weights / sizes 
     -  "Welcome back" = Default (regular weight) 
     -  "Singing in to your account" = should be the size of heading 4 and bold 
     -  "Steps to signing in with GCKey and completing two-factor authentication" should be the size of heading 5 and default size 
     - "Frequently Asked Questions (FAQs)" Should be the size of heading 4 and bold 
     -  The questions in the FAQ should be paragraph size (still in bold)

File: apps/web/src/pages/Auth/SignInPage/SignInPage.tsx [hunk 1]
```diff
@@ -49,141 +57,554 @@ const SignInPage = () => {
...
+          </p>
+          <p>
+            <Link href={loginPath} mode="solid" color="primary" external>
+              {intl.formatMessage({
+                defaultMessage: "Continue to GCKey and sign in",
+                id: "4sLCWZ",
+                description: "GCKey sign in link text on the sign in page",
+              })}
+            </Link>
+          </p>
```
Comment (yonikid15) [inline apps/web/src/pages/Auth/SignInPage/SignInPage.tsx:607]: Needs some padding between the two buttons on mobile view
image

`data-h2-margin-bottom="base(x1)"`

File: apps/web/src/pages/Auth/SignInPage/SignInPage.tsx [hunk 2]
```diff
@@ -49,141 +57,554 @@ const SignInPage = () => {
...
             <p>
-              <Link href={!iapMode ? paths.home() : paths.iap()} mode="inline">
-                {intl.formatMessage({
-                  defaultMessage: "Cancel",
-                  id: "OT0QP3",
+              {intl.formatMessage(
+                {
+                  defaultMessage:
+                    "Read all the FAQ's and still stuck? <helpLink>Contact our team for help</helpLink>",
+                  id: "3M1/4Y",
```
Comment (yonikid15) [inline apps/web/src/pages/Auth/SignInPage/SignInPage.tsx:506]: Can you add a little margin above this message 😅 `data-h2-margin-top="base(x1)"`

**Review by yonikid15 — CHANGES_REQUESTED** [review]
This looks amazing James 🎉 Nice work! I found two small nit picky things, but other then that it looks perfect.

File: apps/web/src/pages/Auth/SignInPage/SignInPage.tsx [hunk 3]
```diff
@@ -49,141 +57,554 @@ const SignInPage = () => {
...
+                  <p>1-800-2318-6290</p>
+                  <p>
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "Customer Service Representatives are available to assist you by phone, year round, 24 hours a day, 7 days a week.",
+                      id: "HqQzy1",
+                      description:
+                        "Sixth answer of the Frequently Asked Questions for logging in - final sentence",
+                    })}
+                  </p>
```
Comment (yonikid15) [inline apps/web/src/pages/Auth/SignInPage/SignInPage.tsx:433]: Can we some margin between each line to match the design like so
image

File: apps/web/src/pages/Auth/SignInPage/SignInPage.tsx [hunk 4]
```diff
@@ -49,141 +57,554 @@ const SignInPage = () => {
...
+                  <p>
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "If you're unsure whether you have an existing GCKey account, continue to the website and try signing in. If you can't remember your password, you can also reset it there.",
+                      id: "n4eLy+",
+                      description:
+                        "Fourth answer of the Frequently Asked Questions for logging in - paragraph 2",
+                    })}
+                  </p>
```
Comment (yonikid15) [inline apps/web/src/pages/Auth/SignInPage/SignInPage.tsx:332]: Can we some margin between each line to match the design like so
image

**Review by yonikid15 — CHANGES_REQUESTED** [review]

File: apps/web/src/pages/Auth/SignInPage/SignInPage.tsx (hunk 1 above)
Comment (vd1992) [reply apps/web/src/pages/Auth/SignInPage/SignInPage.tsx:607]: I think that was already done by Peter in https://github.com/GCTC-NTGC/gc-digital-talent/pull/7835/commits/8fabb396dca6bcbf0dc5e32332e2e73c3034efda
Some tweaks from the sister page PR were rolled into this one

File: apps/web/src/pages/Auth/SignInPage/SignInPage.tsx (hunk 2 above)
Comment (JamesHuf) [reply apps/web/src/pages/Auth/SignInPage/SignInPage.tsx:530]: I've added this here and to the Sign up page.

File: apps/web/src/pages/Auth/SignInPage/SignInPage.tsx (hunk 4 above)
Comment (JamesHuf) [reply apps/web/src/pages/Auth/SignInPage/SignInPage.tsx]: This should be good as well now.

**Review by yonikid15 — APPROVED** [review]
Nice 🥇

**Pull request merged**